src/ringct/rctSigs.cpp

```diff
@@ -163,14 +163,21 @@ namespace rct {
       return verifyBorromean(bb, P1_p3, P2_p3);
     }
 
-    //Multilayered Spontaneous Anonymous Group Signatures (MLSAG signatures)
-    //This is a just slghtly more efficient version than the ones described below
-    //(will be explained in more detail in Ring Multisig paper
-    //These are aka MG signatutes in earlier drafts of the ring ct paper
-    // c.f. https://eprint.iacr.org/2015/1098 section 2. 
-    // Gen creates a signature which proves that for some column in the keymatrix "pk"
-    //   the signer knows a secret key for each row in that column
-    // Ver verifies that the MG sig was created correctly        
+    // Hash a key to p3 representation
+    static void hash_to_p3(const key &k, ge_p3 &hash8_p3) {
```

Comment:
This now aliases a function in `src/ringct/rctOps.cpp` which itself is an alias for a function in `src/crypto/crypto.cpp`.

Merge one of these functions somewhow. The existing one in `src/ringct/rctOps.cpp` can call this function and then immediately convert to bytes (assuming that code somewhere is still using that function).

This can also return-by-value in its current usage, although the benefit seems low since `ge_p1p1_to_p3` takes an out variable anyway.

Reply:
I think this function should be in rctOps and probably replace hashToPoint there.

However, I'm also willing to merge anyway and leave it for the future.

Reply:
Updated. Review requested.